infrastructure/quick-deploy/aws/all/variables.tf

```diff
@@ -110,6 +110,14 @@ variable "eks" {
       scale_down_unneeded_time              = optional(string, "2m")
       skip_nodes_with_system_pods           = optional(bool, true)
       node_selector                         = optional(any, {})
+      version                               = optional(string, "9.24.0")
```

Comment:
Would it make sense to use a similar concept for the helm chart versions than for the docker image versions?

Reply:
We will do this in an other issue.